homeassistant/components/motionblinds_ble/__init__.py

```diff
@@ -98,7 +98,7 @@ def async_update_ble_device(
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
     # Apply options
-    await hass.async_create_background_task(
+    hass.async_create_background_task(
```

Comment:
If the intent is to wait for it, than it should be awaited directly.

If you don't want to wait for it than its fine as-is except please use `entry.async_create_background_task`

Reply:
Sure. I don't think it makes sense to make the setup of the entry wait for the connection of the blind in case a permanent connection is enabled, which is why I use a background task